patches/common/chromium/.patches.yaml

```diff
@@ -477,3 +477,9 @@ patches:
     Pass RenderProcessHost through to PlatformNotificationService.
 
     This is so Electron can identify which renderer a notification came from.
+-
+  author: deepak1556 <hop2deep@gmail.com>
+  file: content_browser_main_loop.patch
+  description: |
+    Pass idle quit closure for main message loop, so that pending tasks are
+    run before shutdown.
```

Comment:
I think we should explicitly note this is required by `CommonWebContentsDelegate::ResetManagedWebContents` , and also have a note in the method's comments that this patch is required.